woocommerce-pdf-invoices-packingslips.php

```diff
@@ -542,6 +545,16 @@ public function legacy_addon_notices() {
 			}
 		}
 	}
+	
+	public function init_number_declaration_error_custom_message( $message, $error ) {
+		if ( isset( $error['message'] ) && false !== strpos( $error['message'], 'Declaration of WPO\WC\PDF_Invoices\Documents\Pro_Document::init_number() must be compatible with' ) ) {
+			$message  = '<p>' . __( 'A plugin extending the document abstract class of the PDF Invoices & Packing Slips for WooCommerce plugin version 3.7.8 or higher may either not be running the latest version or require additional code improvements. Please check the error associated with the affected plugin below.', 'woocommerce-pdf-invoices-packing-slips' ) . '</p>'; 
+			$message .= '<p>' . wp_kses_post( $error['message'] ) . '</p>';
+			$message .= '<p><a href="' . esc_url( '' ) . '">' . __( 'Troubleshoot the issue with this helpful guide!', 'woocommerce-pdf-invoices-packing-slips' ) . '</a></p>';
```

Comment:
We need to create a documentation page to help customers update their Proposal plugin from FTP, because if the site crashes is the only way, I think renaming the plugin doesn't give them access to the backend either.

Reply:
@alexmigf, It seems renaming will work since I could run the site again with just renaming.